example/lib/features/scene/waterfall_flow_fixed_height_demo/waterfall_flow_fixed_height_page.dart

```diff
@@ -0,0 +1,126 @@
+  }
+
+  @override
+  void dispose() {
+    observerController.controller?.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(title: const Text('Waterfall Flow')),
+      body: SliverViewObserver(
+        child: _buildBody(),
+        controller: observerController,
+        autoTriggerObserveTypes: const [
+          ObserverAutoTriggerObserveType.scrollEnd,
+        ],
+        triggerOnObserveType: ObserverTriggerOnObserveType.directly,
+        // extendedHandleObserve: (context) {
+        //   // An extension of the original observation logic.
+        //   final _obj = ObserverUtils.findRenderObject(context);
+        //   if (_obj is RenderSliverWaterfallFlow) {
+        //     return ObserverCore.handleGridObserve(
+        //       context: context,
+        //       fetchLeadingOffset: () => observeOffset,
+        //     );
+        //   }
+        //   return null;
+        // },
+        sliverContexts: () {
+          return [
+            if (grid1Context != null) grid1Context!,
+          ];
+        },
+        onObserveViewport: (result) {
+          firstChildCtxInViewport = result.firstChild.sliverContext;
+          debugPrint(
+              'current first sliver in viewport - $firstChildCtxInViewport');
+        },
+        onObserveAll: (resultMap) {
+          final result = resultMap[firstChildCtxInViewport];
+          debugPrint('all Observe for current first sliver - $result');
+        },
+      ),
+      floatingActionButton: FloatingActionButton(
+        child: const Icon(Icons.airline_stops_outlined),
+        onPressed: () {
+          observerController.jumpTo(
+            index: 13,
+            isFixedHeight: true,
+            renderSliverType: ObserverRenderSliverType.grid,
+          );
+        },
+      ),
+    );
+  }
+
+  Widget _buildBody() {
+    return CustomScrollView(
+      controller: scrollController,
+      slivers: [
+        _buildGridView(childCount: 50),
+      ],
+    );
+  }
+
+  Widget _buildGridView({
+    required int childCount,
+  }) {
+    return SliverWaterfallFlow(
+      gridDelegate: const SliverWaterfallFlowDelegateWithFixedCrossAxisCount(
+        crossAxisCount: 3,
+        mainAxisSpacing: 15,
+        crossAxisSpacing: 10,
+      ),
+      delegate: SliverChildBuilderDelegate(
+        (BuildContext context, int index) {
+          if (grid1Context != context) grid1Context = context;
+          return Container(
+              height: 50,
+              color: Colors.primaries[index % Colors.primaries.length],
+              alignment: Alignment.topLeft,
+              child: Text(
+                'item: $index',
+                style: const TextStyle(fontSize: 20, color: Colors.white),
+              ));
+        },
+        childCount: childCount,
+      ),
+    );
+  }
+
+  Widget _buildSeparator(double size) {
```

Comment:
The declaration '_buildSeparator' isn't referenced.

Reply:
Deleted